apps/dashboard/.env

```diff
@@ -1,5 +1,5 @@
 # REPLACE THIS IT IS FOR TESTING
-NEXTAUTH_URL=https://c890-2405-201-301c-5b67-89d6-bd56-6afb-6294.ngrok-free.app
+NEXTAUTH_URL=https://39a0-2405-201-301c-5b67-8cea-1030-d56-e0b3.ngrok-free.app
```

Comment:
we should not change this for every PR this is creating noise for the reviewer.

it's actually possible to have `localhost` here. look at my PR if that's useful: https://github.com/GaloyMoney/galoy/pull/3407/files

the reason I stumble upon some issues at the beginning was because there were a mix of `127.0.0.1` and `localhost`.